config/routes.rb

```diff
@@ -25,6 +27,7 @@
     resources :proposed_organisation_edits, :only => [:new, :show, :create, :update]
   end
   resources :users
+  resources :events, :only => [:new, :create, :show, :index, :update, :destroy]
```

Comment:
We can as well omit the only section since you are allowing every action of the controller